pubnub-users/build.gradle

```diff
@@ -1,31 +1,4 @@
-plugins {
-    id 'org.jetbrains.kotlin.jvm'
-    id 'org.unbroken-dome.test-sets'
-}
-
-apply plugin: "com.vanniktech.maven.publish"
-
-repositories {
-    mavenCentral()
-}
-
-testSets {
-    integrationTest
-}
-
 dependencies {
     implementation project(":")
```

Comment:
What about an idea of creating pubnub-core module? Isn't it right time to do this?

Reply:
I don't know. Is it the right time to do?

Reply:
My understanding is that this change will not be visible to clients. They would configure include our SDK the same way as today. In such case I think that this might be good right time. We can take this off line.

Reply:
Yes, it would not change a thing from the users perspective